src/tqec/plaquette/plaquette.py

```diff
@@ -44,6 +58,18 @@ def qubits(self) -> list[PlaquetteQubit]:
     def circuit(self) -> ScheduledCircuit:
         return self._circuit
 
+    def _caclulate_origin(self, origin: PlaquetteOrigin, origin_index: int) -> None:
+        """Calcluates the origin of the plaquette given the user input."""
+        match origin:
+            case PlaquetteOrigin.TOP_LEFT:
+                self._origin = Position(0, 0)
+            case PlaquetteOrigin.CENTRAL:
+                raise NotImplementedError()
+            case PlaquetteOrigin.FIRST:
+                self._origin = self._qubits[0].position
+            case PlaquetteOrigin.USER_DEFINED:
+                self._origin = self._qubits[origin_index].position
+
```

Comment:
Following another comment, if the origin is implicitly defined by the user input (i.e., always `(0, 0)`), this method may not be needed.